src/FluentValidation/Internal/MessageFormatter.cs

```diff
@@ -93,21 +92,26 @@ public MessageFormatter AppendPropertyValue(object value)
 		/// </summary>
 		public Dictionary<string, object> PlaceholderValues => _placeholderValues;
 
-		protected virtual string ReplacePlaceholderWithValue(string template, string key, object value) {
-			string placeholder =  GetPlaceholder(key);
-			return template.Replace(placeholder, value?.ToString());
-		}
+		// TODO: Add alignment component
+		private Regex _keyRegex = new Regex("{([^{}:]+)(?::([^{}]+))?}"); // Matches any placeholder

```

Comment:
For performance, might be better for this to be static and to use `RegexOptions.Compiled`

Reply:
Cannot use Compiled as fluentvalidation dependency netstandard1.1 don't have it.

Reply:
Correct, but you can use it with the other target frameworks using conditional compilation, eg:

```
#if NETSTANDARD1_1 || NETSTANDARD1_6
private Regex _keyRegex = new Regex("{([^{}:]+)(?::([^{}]+))?}");
#else
private Regex _keyRegex = new Regex("{([^{}:]+)(?::([^{}]+))?}", RegexOptions | Compiled);
#endif

Let's add this please.

Reply:
I'm having trouble making the regex stuff compatible with all the frameworks. net45 and netstandard2.0 doesn't support it.